tools/WinObjC.Tools/vsimporter-templates/packageable.project.json

```diff
@@ -0,0 +1,21 @@
+{
+  "dependencies": {
+    "WinObjC.Packaging": "*",
```

Comment:
This mostly a question of what it looks like to invoke vsimporter on middleware VS an app. How does vsimporter know it's middleware? Do we need more user obvious knobs to control this?